app/utils/prepareAnswer.js

```diff
@@ -82,6 +93,24 @@ function prepareStatsDistrictAnswer(body, stateName) {
     return data
 }
 
+function prepareStateTestStat(body, stateName) {
+
+    let stateWise = body['states_tested_data'], stateDataIndex = -1
+    for (let index = 0; index < stateWise.length; index++) {
+        let currentData = stateWise[index]
+        if (currentData['state'].toLowerCase() === stateName.toLowerCase())
+            stateDataIndex = index
+    }
+    if (stateDataIndex === -1)
+        return '\n\u{26AA} Citizens Tested: Not Available'
```

Comment:
In this case, we will show nothing about testing data. No need to show this message.

Reply:
okay will implement this too 💯

Reply:
done 👍